pkg/storage/client_raft_test.go

```diff
@@ -1622,18 +1622,8 @@ func TestReportUnreachableHeartbeats(t *testing.T) {
 	mtc := startMultiTestContext(t, 3)
 	defer mtc.Stop()
 
-	// Replicate the range onto all three stores
 	mtc.replicateRange(1, 1, 2)
 
-	// Send a single increment to ensure the range is up.
```

Comment:
Yeah, there's a difference between the replica being present and the replica having fully applied the log. Some tests rely on this; in others it's just copy-pasta. Here I think it's the latter.